primitives/src/config/asset_runtime_config.rs

```diff
@@ -15,7 +15,7 @@ use crate::{
 use codec::{Decode, Encode};
 use core::fmt::Debug;
 
-/// Default set of commonly used types by Substrate kitchensink runtime.
+/// Commonly used config of Substrate runtimes that use the asset pallet.
```

Comment:
Maybe "Standard runtime config for Substrate and Polkadot nodes that use the asset pallet" ?